components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1431,12 +1432,40 @@ void RewardsServiceImpl::SetCatalogIssuers(const std::string& json) {
   bat_ledger_->SetCatalogIssuers(json);
 }
 
+std::vector<uint64_t> RewardsServiceImpl::GetEarningsRange() {
+  uint64_t difference;
+  auto base_time_now = base::Time::Now();
+  int one_day = (60 * 60 * 24);
+
+  base::Time::Exploded exploded;
+  base_time_now.LocalExplode(&exploded);
+
+  if (exploded.month > 5) {
```

Comment:
should this not be exploded.day?

Reply:
woops, yes it should

Reply:
and looks like it's `day_of_month` https://chromium.googlesource.com/chromium/chromium/+/master/base/time.h#218